### PR TITLE
Impl Debug Trait for Snapshot

### PR DESCRIPTION
**Pull request by breeswish**
Enables using `#[derive(Debug)]` when the structure contains a `Snapshot` field.

@BusyJay @hicqu PTAL

### CODE REVIEW

**Comment by hicqu** [comment]
LGTM.

File: src/raftstore/store/snap.rs [hunk 1]
```diff
@@ -817,6 +817,12 @@ fn apply_plain_cf_file<D: CompactBytesDecoder>(
     Ok(())
 }
 
+impl fmt::Debug for Snap {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        write!(f, "Snap")
```
Comment (BusyJay) [inline src/raftstore/store/snap.rs]: This is too simple as a debug message for `Snap`.

Comment (breeswish) [reply src/raftstore/store/snap.rs]: Updated. PTAL

**Comment by BusyJay** [comment]
LGTM, please resolve the conflict.

**Comment by siddontang** [comment]
Any update @breeswish

**Comment by breeswish** [comment]
@BusyJay @siddontang  Merged master, PTAL again.

**Comment by BusyJay** [comment]
ci fail

**Pull request merged**